### PR TITLE
docs

### PR DESCRIPTION
**Pull request by probablyArth**
Will be adding more commits for further docs

### CODE REVIEW

File: src/account/useEmailSignIn.ts [hunk 1]
```diff
@@ -8,7 +8,10 @@ type TRequest = {
   email: string,
   password: string,
 }
-
+/**
+ * Create email session
+ * {@link https://appwrite.io/docs/client/account?sdk=web-default#accountCreateEmailSession | create email session documentation}
```
Comment (sanny-io) [inline src/account/useEmailSignIn.ts:13]: This is currently how it appears in VS Code.



It should look like this (I'm using another function as an example).



https://github.com/sanny-io/react-appwrite/blob/3f160484efaaf71395a1d30181f1ea5b8d4b7069/src/databases/useDocumentUpdate.ts#L16

**Review by sanny-io — CHANGES_REQUESTED** [review]

File: src/account/useEmailSignIn.ts (hunk 1 above)
Comment (probablyArth) [reply src/account/useEmailSignIn.ts:13]: Alright, got it 👌

File: src/account/useEmailSignIn.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ type TRequest = {
 }
 /**
  * Create email session
- * {@link https://appwrite.io/docs/client/account?sdk=web-default#accountCreateEmailSession | create email session documentation}
+ * @link [appwrite create email session documentation](https://appwrite.io/docs/client/account?sdk=web-default#accountCreateEmailSession)
```
Comment (sanny-io) [inline src/account/useEmailSignIn.ts:13]: This is close. I should have been clearer, my apologies. It should just say `Appwrite Documentation` between the square brackets.

**Review by sanny-io — CHANGES_REQUESTED** [review]

**Review by sanny-io — APPROVED** [review]
There we go 🥳

This is the way! Keep committing and @mention me again when ready to merge it all 🙂

Thank you bro 🙏

**Comment by probablyArth** [comment]
@sanny-io I think that's all the documentation for now.

**Review by sanny-io — APPROVED** [review]
Looks good! Well done bro.

**Pull request merged**